autorag/utils/preprocess.py

```diff
@@ -44,6 +44,7 @@ def cast_generation_gt(gt):
         else:
             raise ValueError(f"generation_gt must be str or list, but got {type(gt)}")
 
+    df = df.reset_index()
```

Comment:
You must set `drop=True` at reset_index.

When you see the result dataframe at test file, you can find new `index` column like this.

![image](https://github.com/Marker-Inc-Korea/AutoRAG/assets/28955029/59a2b9aa-8b2d-418a-848f-6f0ab5ee5f6a)